requirements.txt

```diff
@@ -1,36 +1,12 @@
 # The single source of truth for the OpenSpiel pip dependencies.
```

Comment:
Not a single source of truth anymore.

Reply:
Same reason as above, the packages in python_extra_deps are truly no longer official dependencies. The idea is that official dependencies in this file reflects only what is necessary to run the core python API.

Reply:
I probably will rephrase "single source of truth" anyway because that feels too strong now.